src/vendor_upstream/dom/DOMMouseMoveTracker.js

```diff
@@ -71,11 +70,6 @@ class DOMMouseMoveTracker {
         'mouseleave',
         this._onMouseEnd
       );
-      this._eventOutToken = EventListener.listen(
```

Comment:
What's the reason for removing this rather than just fixing it?

It was added to fix #104, could you test out those cases on a few browsers?  I tried in Chrome & Firefox and it seems the mouseleave event is enough for the fix to hold.  @sisyphean2 do you remember why you added both mouseleave & mouseout handlers?  Are there cases we should test?

Reply:
Note that because the mouseout hander was misspelled, (this.onMouseEnd instead of this._onMouseEnd) it never did anything. Fixing the spelling makes it fail miserably. No mouseout handler is needed.